resources/lib/tubecast/youtube/app.py

```diff
@@ -241,7 +241,15 @@ def handle_cmd(self, cmd):
                 self.cur_list = video_ids.split(",")
                 # Set info on our custom player instance and request playback
                 self.player.setInfo(cur_video_id, self.ctt, self.cur_list_id, self.current_index)
-                self.player.play_from_youtube(kodibrigde.get_youtube_plugin_path(cur_video_id))
+
+                try:
```

Comment:
use `data.get("currentTime")` and threat the received value instead of try/excepting please

Reply:
I was adhering to the [EAFP](https://docs.python.org/3/glossary.html#term-eafp) style here but I have absolutely no problem changing it.